src/pymmcore_widgets/hcs/_hcs_wizard.py

```diff
@@ -0,0 +1,241 @@
+
+        self.plate_page = _PlatePage(self)
+        self.calibration_page = _PlateCalibrationPage(self._mmc, self)
+        self.points_plan_page = _PointsPlanPage(self._mmc, self)
+
+        self.addPage(self.plate_page)
+        self.addPage(self.calibration_page)
+        self.addPage(self.points_plan_page)
+
+        # SAVE/LOAD BUTTONS ----------------------
+
+        # add custom button to save
+        self.setOption(QWizard.WizardOption.HaveCustomButton1, True)
+        if save_btn := self.button(QWizard.WizardButton.CustomButton1):
+            save_btn.setText("Save")
+            save_btn.clicked.connect(self.save)
+            self.setButton(QWizard.WizardButton.CustomButton1, save_btn)
+        # add custom button to load
+        self.setOption(QWizard.WizardOption.HaveCustomButton2, True)
+        if load_btn := self.button(QWizard.WizardButton.CustomButton2):
+            load_btn.setText("Load")
+            load_btn.clicked.connect(self.load)
+            self.setButton(QWizard.WizardButton.CustomButton2, load_btn)
+
+        # CONNECTIONS ---------------------------
+
+        self.plate_page.widget.valueChanged.connect(self._on_plate_changed)
+        self._on_plate_changed(self.plate_page.widget.value())
+        self.calibration_page.widget.calibrationChanged.connect(
+            self._on_calibration_changed
+        )
+
+    def sizeHint(self) -> QSize:
+        return QSize(880, 690)
+
+    def value(self) -> useq.WellPlatePlan | None:
+        """Return the current well plate plan, or None if the plan is uncalibrated."""
+        calib_plan = self.calibration_page.widget.value()
+        if not self._calibrated or not calib_plan:
+            return None
+
+        plate_plan = self.plate_page.widget.value()
+        if plate_plan.plate != calib_plan.plate:
+            warnings.warn("Plate Plan and Calibration Plan do not match.", stacklevel=2)
+            return None
+
+        return useq.WellPlatePlan(
+            plate=plate_plan.plate,
+            selected_wells=plate_plan.selected_wells,
+            rotation=calib_plan.rotation,
+            a1_center_xy=calib_plan.a1_center_xy,
+            well_points_plan=self.points_plan_page.widget.value(),
+        )
+
+    def setValue(self, value: useq.WellPlatePlan) -> None:
+        """Set the state of the wizard to a WellPlatePlan."""
+        self.plate_page.widget.setValue(value)
+        self.calibration_page.widget.setValue(value)
+        self.points_plan_page.widget.setValue(value.well_points_plan)
+
+    def save(self, path: str | None = None) -> None:
+        """Save the current well plate plan to disk."""
+        if not isinstance(path, str):
+            path, _ = QFileDialog.getSaveFileName(
+                self,
+                "Save Well Plate Plan",
+                "",
+                "JSON or YAML (*.json *.yaml *.yml)",
+            )
+        if path:
+            value = self.value()
+            if value is None:
+                return
```

Comment:
this is somewhat similar to the concept of disabling the Finish button if the plate is uncalibrated.  If you don't want the save button to do anything when the plate is not yet calibrated, then that button *must* be disabled until it's valid to click it.   It's not ok to let them click on it, pick a path, and then have it just close silently without doing anything